sanity.json

```diff
@@ -1,84 +1,84 @@
-    "GraphQL API": "No",
+    "GraphQL API": "Yes",
     "GraphQL [Mutations]": "No",
     "GraphQL [Subscriptions]": "No",
-    "REST API": "Yes",
+    "REST API": "Yes, GROQ",
     "REST [Create]": "Yes",
     "REST [Read]": "Yes",
     "REST [Update]": "Yes",
     "REST [Delete]": "Yes",
-    "REST [Upload]": "No",
+    "REST [Upload]": "Yes",
     "Search API": "Yes",
-    "Search Features [Full Text Search]": "No",
+    "Search Features [Full Text Search]": "Yes",
     "Search Features [Stemming]": "No",
     "Search Features [Stop Words]": "No",
     "Search Features [Boosting]": "No",
-    "Search Features [Autocompletion]": "No",
-    "Search Features [Autosuggestion]": "No",
-    "Search Features [Result Highlighting]": "No",
+    "Search Features [Autocompletion]": "NA",
+    "Search Features [Autosuggestion]": "NA",
+    "Search Features [Result Highlighting]": "NA",
     "Search Features [Geospatial Search]": "No",
     "Search Features [Search within uploads (pdf,doc)]": "No",
     "Image Manipulation": "Yes",
-    "Image Focalpoint support": "",
-    "Image Facedetection": "",
+    "Image Focalpoint support": "Yes",
+    "Image Facedetection": "No",
     "Optimized Image Encoding": "Yes",
-    "Asset fingerprinting": "",
+    "Asset fingerprinting": "No",
     "CDN Support": "Yes",
     "Antivirus Scanning": "No",
-    "Custom Binary Handler": "No",
+    "Custom Binary Handler": "Yes",
     "System Requirements": "NA",
     "Supported Databases": "NA",
-    "Clustering": "No",
+    "Clustering": "NA",
     "Docker Support": "NA",
-    "Backup Feature": "No",
+    "Backup Feature": "Yes",
     "Import/Export": "Yes",
     "CLI": "Yes",
     "SDK [Java]": "No",
-    "SDK [C#]": "No",
+    "SDK [C#]": "Yes",
     "SDK [PHP]": "Yes",
     "SDK [JavaScript]": "Yes",
-    "SDK [React]": "No",
+    "SDK [React]": "Yes",
     "SDK [AngularJS]": "No",
-    "SDK [TypeScript]": "No",
+    "SDK [TypeScript]": "Yes",
     "Web Hooks": "Yes",
     "Eventbus": "Yes",
     "Bulk Import": "Yes",
-    "Client Side Forms": "No",
+    "Client Side Forms": "Yes",
     "Plugin System": "Yes",
     "Customizable UI": "Yes",
-    "Custom Field Types": "No",
-    "User Management": "",
+    "Custom Field Types": "Yes",
+    "User Management": "Yes",
     "Role Based Permissions": "Yes",
     "Document Level Permissions": "Yes",
-    "OAuth 2.0 Support": "No",
-    "Auditing": "",
+    "OAuth 2.0 Support": "Yes",
+    "Auditing": "Yes",
     "API Keys": "Yes",
     "Project Support": "Yes",
     "I18N Localized Content": "Yes",
-    "Content Trees": "",
-    "Tagging": "",
-    "Content Relations": "",
-    "Nesting of Fields": "",
-    "GDPR Statement": "",
-    "GDPR API": "No",
-    "Editing Conflict Detection": "",
+    "Content Trees": "Yes",
+    "Tagging": "Yes",
+    "Content Relations": "Yes",
+    "Nesting of Fields": "Yes",
+    "GDPR Statement": "Yes",
+    "GDPR API": "Yes",
+    "Editing Conflict Detection": "Yes",
```

Comment:
Backend is real-time, every patch get a revision ID

Reply:
What happens when a content is updated with two different requests? Will the second request override the updated data of the first request?

Reply:
Everything is transactional, so if the transaction includes the revision id of the document, it will fail if the document has been changed meanwhile (and tell the client). If not, it will update the document.

Reply:
@kmelve Very nice. Is this documented somewhere? What is the error code in those cases? 409? 400?